weblate/trans/vcs.py

```diff
@@ -732,8 +732,11 @@ def push(self):
         Forks repository on Github, pushes changes to *-weblate branch
         on fork and creates pull request against original repository.
         """
-        self.fork()
         fork_branch = '{0}-weblate'.format(self.branch)
```

Comment:
maybe a change in just to this line, to something like

```
fork_branch = '{0}-weblate'.format(self.name)
```

I don't know the code so much to guess if this would put the subproject name instead of the branch name.